README.md

```diff
@@ -27,8 +30,78 @@ config = {
     'region': 'us-east-2',
     'subscriber': '00000000000',       
     'name': 'profile',
-    'env': 'prod'
+    'env': 'dev'
+}
+
+publisher = get_publisher(config=config, pub_type='sns')
+print(publisher.arn)
+>>> 'arn:aws:sns:us-east-2:00000000000:profile--dev'
+```
+
+Publish:
+
+```python
+from elife_bus_sdk.events import Event
+
+publisher.publish(Event(foo='bar', some='fields'))
+
+```
+
+Queue
+---------
+
+Current supported:
+- [SQS](https://aws.amazon.com/sqs/)
+
+
+Configuration:
+
+```python
+from elife_bus_sdk import get_queue
+
+config = {   
+    'QueueName': 'some_queue',
 }
 
-publisher = get_publisher(pub_name='test', pub_type='sns', config=config)
+message_queue = get_queue(config=config, pub_type='sqs')
 ```
+
+Poll for messages:
+```python
+for msg in message_queue.poll():
+    print(msg)
+```
+
+Receive message(s):
+```python
+msg = message_queue.dequeue()
+```
+
+Send message:
+```python
+message_queue.enqueue('test message')
+```
```

Comment:
A link to `boto3` can be added explaining AWS credentials have to be provided in a way that it supports, either through environment variables or through the filesystem on `~/.aws/credentials`